src/layouts/default.html.eco

```diff
@@ -64,7 +64,7 @@
 					<% end %>
 					<li>
 						<a href="//discourse.transformap.co">
-							Forum
+							Agora
```

Comment:
There was a 4:4:4 EQUAL vote for Forum, Discourse and Community. Forum isn't out, so who is about to decide?